### ISSUE TITLE
setConfigPortalBlocking and additional parameter portail not compatible 

### ISSUE THREAD

**Issue by spilz87** [tzapu/WiFiManager#659]
I'm not able to use setConfigPortalBlocking(false) and add parameters on portail on the same project, I tried to merge the 2 exemples and each time I got a reset loop 

Some one as succed on it or is it an issue ?

**Comment by tablatronix** [comment]
is this still an issue?

**Comment by spilz87** [comment]
For me yes it is, I'm not able to merge the both exemples

**Comment by tablatronix** [comment]
do you have the latest commits?

**Comment by spilz87** [comment]
I download the last development branch last week end

**Comment by tablatronix** [comment]
It has been updated since then

**Comment by spilz87** [comment]
Ok sorry 
I will download the new one and try it again

**Comment by spilz87** [comment]
I will let you know

**Comment by irwanayeah** [comment]
hello @ spilz87 @tablatronix , is there a solution? because I got the same thing

I use the sample WifiManager \ examples \ DEV \ OnDemandConfigPortal \ OnDemandConfigPortal.ino

then add lines
setConfigPortalBlocking (false); on setup
process (); on the loop

parameters never seen.

**Comment by tablatronix** [comment]
Your parameters are setup in setup then they are out of scope in loop. You have to put then somewhere else or make them static or something

**Comment by irwanayeah** [comment]
work perfect. thank @tablatronix

**Comment by tablatronix** [comment]
Yeah that is one of the problems with the non blocking  and has to be documented and a new example made for it, unless we can make it better, like store parm objects in wm.
Then there is the issue of cleaning up when its global etc.

Ideas welcome

**Comment by irwanayeah** [comment]
hi @tablatronix , how about function setSaveConfig Callback(save Config Callback); never fires after saving value of parameters.

**Comment by tablatronix** [comment]
Yeah I keep forgetting to work on that. Can you create an issue so I dont forget again.

Since params can now be saved seperate from wifi I need to add new callbacks also

**Comment by irwanayeah** [comment]
ok
